README.md

```diff
@@ -1,151 +1,119 @@
-
-### Documentation
-
-- [wdl-aid](https://github.com/biowdl/wdl-aid) generates documentation for the inputs of WDL workflows, based on the parameter_meta information defined in the WDL file.	
-- [wdldoc](https://github.com/stjudecloud/wdldoc)
-
-### Testing
-
-- [wdl-tests](https://github.com/openwdl/wdl-tests) is a collection of test cases for WDL implementations. A specification is provided for writing new tests that are compatible with automated testing frameworks.
-- [Pytest-workflow](https://github.com/LUMC/pytest-workflow) is a implementation-agnostic workflow tester. It can be used with both Cromwell and MiniWDL. Uses pytest as the underlying test framework. Tests can be specified in either YAML format or python code.
-- [Pytest-wdl](https://github.com/EliLillyCo/pytest-wdl) is a plugin for the pytest unit testing framework that enables testing of WDL tasks and workflows. Tests can be specified in either YAML format or python code.
-
-### Packaging
-
-- [miniwdl zip](https://miniwdl.readthedocs.io/en/latest/zip.html) generates a ZIP file including a given WDL source code file and any other WDL files it imports. The ZIP file can be supplied directly to miniwdl run, which can extract it automatically.
-- [wdl-packager](https://github.com/biowdl/wdl-packager) packages a workflow and all of its imports into a zip file. The zip can be used as an imports zip package for cromwell. The utility can add non-WDL files (such as the license) to the zip package and provides options to package the zip in a binary reproducible way.
-
-## Contributing
-
-WDL only advances through community contributions.
-While participating in discussions and submitting issues are great ways to be involved, help is also needed to implement changes to the specification.
-For more information on how you can contribute, please read the [Contributing](CONTRIBUTING.md) guide.
-
-### RFC Process
-
-Submitted [pull requests](https://github.com/openwdl/wdl/pulls) are subject to the [RFC Process](RFC.md).
-Please review and familiarize yourself with the process if you would like to see changes submitted to the specification.
-
-## Governance
-
-The WDL specification is entirely community driven; however, it is overseen by a [Governance committee](GOVERNANCE.md).
-If you are interested in being involved in WDL governance, please join the [Slack](https://join.slack.com/t/openwdl/shared_invite/zt-ctmj4mhf-cFBNxIiZYs6SY9HgM9UAVw) and post a message in the `#general` channel.
+<div style="align: center">
+  <img src="https://raw.githubusercontent.com/claymcleod/wdl/refs/heads/docs/readme-update/.github/assets/readme-header.png">
+</div>
+
+<br />
+
+The **Workflow Description Language (WDL)** (pronounced as _/hwɪdl/_ or "whittle" with a
+'d') is an open standard for describing data processing workflows using a
+human-readable/writeable syntax. It introduces a domain-specific language that aims to
```

Comment:
We adopted the convention of one sentence per line to make PRs smaller/reviews easier.

Reply:
Sounds good, will do.